package.json

```diff
@@ -14,6 +14,7 @@
     "dotenv": "^5.0.1",
     "express": "^4.16.2",
     "express-hbs": "^1.0.4",
+    "git-rev-sync": "^1.12.0",
```

Comment:
I think we're missing updates to the package-lock.json file if we're adding a module, no?

Reply:
Added `package-lock.json` to this PR.